timesketch/lib/datastores/elastic.py

```diff
@@ -328,6 +335,7 @@ def set_label(
             script_name = u'toggle_label'
         script = {
             u'script': {
+                u'lang': u'groovy',
```

Comment:
No idea how this will behave if you copy two scripts that have the same name. It will probably depend on how ES is configured (groovy or painless) but IIRC ES5 uses painless by default so you might want to tweak the config there.